### PR TITLE
Update based on Feedback from TME

### PR DESCRIPTION
**Pull request by mengdong**
2 fixes in README to resolve startup error user might run into

### CODE REVIEW

**Comment by CoderHam** [comment]
@mengdong these changes look good to me. Have they been tested?

**Comment by mengdong** [comment]
Hi Hemant, I marked this as WIP now, TME team will add additional feedback they have.

File: deploy/gke-marketplace-app/README.md [hunk 1]
```diff
@@ -60,15 +60,16 @@ Currently, GKE >= 1.18.7 only supported in GKE rapid channel, to find the latest
...
 --addons=HorizontalPodAutoscaling,HttpLoadBalancing,Istio \
 --machine-type=n1-standard-8 \
 --node-locations=${ZONE} \
 --subnetwork=default \
 --scopes cloud-platform \
---num-nodes 1
+--num-nodes 1 \
+--project ${PROJECT_ID}
 
 # add GPU node pools, user can modify number of node based on workloads
```
Comment (AshishSardana) [inline deploy/gke-marketplace-app/README.md:74]: @mengdong Creating a cluster and adding GPU nodes could take up-to 10 minutes. Please be patient after executing this command.

Comment (mengdong) [reply deploy/gke-marketplace-app/README.md:75]: added

**Review by mengdong — COMMENTED** [review]
address TME team's review.

**Comment by mengdong** [comment]
@CoderHam I realize this has been closed. Do I need to rebase from master or the pull request is good to merged, and yes we have tested this.

**Comment by CoderHam** [comment]
@mengdong yes please rebase from master. Thanks for the update. 
cc @rmccorm4 in case you have any feedback

**Comment by mengdong** [comment]
@CoderHam I just rebased from master, mostly TME feedback so nothing substantial

**Comment by rmccorm4** [comment]
> cc @rmccorm4 in case you have any feedback

LGTM 👍

**Review by CoderHam — APPROVED** [review]

**Pull request merged**